AWSiOSSDKv2.xcodeproj/project.pbxproj

```diff
@@ -397,10 +397,29 @@
 		B4A4E04022B4281000379396 /* libOCMock.a in Frameworks */ = {isa = PBXBuildFile; fileRef = CEB8EF551C6A6A2E0098B15B /* libOCMock.a */; };
```

Comment:
I'm not exactly sure where in the .pbxproj file it's located, but please alphabetize:

- The SDKs listed in the Project Inspector
- The targets (Framework & tests) in the targets list

Reply:
Sorted them.